{"uid":"c97fca479371912b","name":"test_update_backup_password","fullName":"tests.integration.backup_tests.test_backups#test_update_backup_password","historyId":"84b7ed0c6d51f85869e9196a8db97b92","time":{"start":1728352540626,"stop":1728352540626,"duration":0},"description":"Verifies that after changing the backup password the pbm tool is updated and functional.","descriptionHtml":"<p>Verifies that after changing the backup password the pbm tool is updated and functional.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 842, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 874, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 696, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 531, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 616, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1090, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728351332107,"stop":1728351332108,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728351332108,"stop":1728351332375,"duration":267},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728351332108,"stop":1728351332108,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728352540626,"stop":1728352540626,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Verifies that after changing the backup password the pbm tool is updated and functional.","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 842, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 874, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 696, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 531, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 616, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1090, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/mongodb-k8s-operator/mongodb-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"ec2999152d1a177a","name":"log","source":"ec2999152d1a177a.txt","type":"text/plain","size":1126}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1728352540627,"stop":1728352540943,"duration":316},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728352540944,"stop":1728352540944,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728352540944,"stop":1728352540944,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728352540944,"stop":1728352540945,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.backup_tests"},{"name":"suite","value":"test_backups"},{"name":"host","value":"fv-az1020-206"},{"name":"thread","value":"13203-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.backup_tests.test_backups"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"36c1cb3c1a6a684","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":21,"passed":34,"unknown":1,"total":56},"items":[{"uid":"abae2f4c453c1078","reportUrl":"../2484//#testresult/abae2f4c453c1078","status":"passed","time":{"start":1728267669054,"stop":1728267721429,"duration":52375}},{"uid":"47c26c0c05c6d12c","reportUrl":"../2483//#testresult/47c26c0c05c6d12c","status":"passed","time":{"start":1728181676596,"stop":1728181728307,"duration":51711}},{"uid":"cd9f4009c0770cc1","reportUrl":"../2481//#testresult/cd9f4009c0770cc1","status":"passed","time":{"start":1728008311656,"stop":1728008363398,"duration":51742}},{"uid":"ed93e892786b45e9","reportUrl":"../2477//#testresult/ed93e892786b45e9","status":"passed","time":{"start":1727922433960,"stop":1727922485680,"duration":51720}},{"uid":"63ee6f9827123eb8","reportUrl":"../2465//#testresult/63ee6f9827123eb8","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727834395739,"stop":1727834395739,"duration":0}},{"uid":"296bd8b537f2a0ad","reportUrl":"../2456//#testresult/296bd8b537f2a0ad","status":"passed","time":{"start":1727749542920,"stop":1727749594606,"duration":51686}},{"uid":"52e8cd82cfc297fd","reportUrl":"../2444//#testresult/52e8cd82cfc297fd","status":"unknown","time":{}},{"uid":"47d9b8e2d37d1591","reportUrl":"../2443//#testresult/47d9b8e2d37d1591","status":"passed","time":{"start":1727576167367,"stop":1727576552496,"duration":385129}},{"uid":"c9cb416bf00d355","reportUrl":"../2438//#testresult/c9cb416bf00d355","status":"passed","time":{"start":1727403223920,"stop":1727403275643,"duration":51723}},{"uid":"a1c61cc7db681fd1","reportUrl":"../2434//#testresult/a1c61cc7db681fd1","status":"passed","time":{"start":1727317082857,"stop":1727317134594,"duration":51737}},{"uid":"c8bbd1bd6fb654cb","reportUrl":"../2429//#testresult/c8bbd1bd6fb654cb","status":"passed","time":{"start":1727230870829,"stop":1727230922587,"duration":51758}},{"uid":"3b2bc18a4fa6c37e","reportUrl":"../2421//#testresult/3b2bc18a4fa6c37e","status":"passed","time":{"start":1727145063810,"stop":1727145115472,"duration":51662}},{"uid":"4e03bfc3bca1dd52","reportUrl":"../2357//#testresult/4e03bfc3bca1dd52","status":"passed","time":{"start":1726193610406,"stop":1726193973235,"duration":362829}},{"uid":"1effc345d2468674","reportUrl":"../2348//#testresult/1effc345d2468674","status":"passed","time":{"start":1726107447498,"stop":1726107499170,"duration":51672}},{"uid":"4ac99e7c8fef4bbe","reportUrl":"../2330//#testresult/4ac99e7c8fef4bbe","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726019620377,"stop":1726019620377,"duration":0}},{"uid":"d395f82b38d3da81","reportUrl":"../2317//#testresult/d395f82b38d3da81","status":"passed","time":{"start":1725847956840,"stop":1725848289249,"duration":332409}},{"uid":"2d47ad6ca8bbbff3","reportUrl":"../2316//#testresult/2d47ad6ca8bbbff3","status":"passed","time":{"start":1725761525610,"stop":1725761578279,"duration":52669}},{"uid":"67dbf5c7496bc11e","reportUrl":"../2315//#testresult/67dbf5c7496bc11e","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725674114216,"stop":1725674114216,"duration":0}},{"uid":"85801ce5bede2c39","reportUrl":"../2312//#testresult/85801ce5bede2c39","status":"passed","time":{"start":1725588830912,"stop":1725589185601,"duration":354689}},{"uid":"84e55fa2d6f3ec09","reportUrl":"../2309//#testresult/84e55fa2d6f3ec09","status":"passed","time":{"start":1725502703941,"stop":1725502755639,"duration":51698}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"c97fca479371912b.json","parameterValues":[]}